internal/config/migration/migration.go

```diff
@@ -28,11 +29,11 @@ const (
 // Config defines configuration used by CLI.
 type Config struct {
 	Version             string         `yaml:"version" validate:"eq=v2"`
-	Image               string         `yaml:"image" validate:"required,version_compatibility"`
+	Image               string         `yaml:"image" validate:"required,image_compatibility"`
```

Comment:
I think you can just ignore the validation tags here.
They are mostly leftover because I simply copied the structs from the previous Config definition.